src/choose.py

```diff
@@ -91,4 +91,8 @@ def setSelectionsFromPickle(selectionPath, lineObjs):
         output.writeToFile('echo ":D"')
         sys.exit(0)
     output.clearFile()
-    curses.wrapper(doProgram)
+    # we initialize our args *before* we curse up
+    # so we can benefit from the default argparse
+    # behavior:
+    flags = ScreenFlags.initFromArgs(sys.argv)
+    curses.wrapper(functools.partial(doProgram, flags=flags))
```

Comment:
that's cleaner, I was afraid because I didn't know what python level supported what.